CDCgov/prime-reportstream
# 2985 LIVD lookup with model value variation

This PR adds the ability to the LIVD lookup to use a variation on the model value being used for the lookup.  See #2985 for more information, but in summary, some models have an * at the end and some not and the senders sometimes add or remove those incorrectly.

Test Steps:
1. Run test unit and integration tests and verify they pass.
2. Run the RS functions
3. Make a copy of `src/testIntegration/resources/datatests/CSV_to_HL7/sample-single-pdi-20210608-0002.csv` and add an * at the end of the instrument ID of `BinaxNOW COVID-19 Ag Card` so it reads `BinaxNOW COVID-19 Ag Card*`.  Also delete the ordered test code of `94558-4`.
4. Send this file to RS using the client name `simple_report`
5. Wait until the system processes the file.
6. Open the latest `pima-az-covid` file in your build/sftp folder and verify the ordered test name is populated.

## Changes
- LIVD lookup adds or remove a * from a model value to try to get a match

## Checklist

### Testing

### Process

## Fixes
- #2985

## To Be Done

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
